docs/playbook-reference/defining-playbooks/playbook-basics.rst

```diff
@@ -88,6 +88,43 @@ In order to override the default configuration of the same playbook, both disabl
       - image_pull_backoff_reporter: {}
 
 
+Organizing Playbooks
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Using ``namedCustomPlaybooks``, you can define groups of playbooks separated by cluster, categories, team/department, etc.
+
+They are all merged together into a single playbooks list. This allows you to split away the custom playbooks from ``generated_values.yaml`` to separate files and organize your playbooks.
+
+First, add the custom playbooks as a dictionary into a file named ``app_a_playbooks.yaml`` as shown below:
+
+
+.. code-block:: yaml
+
+    namedCustomPlaybooks:
+    team-a-app-a:
+      - triggers:
+          - on_prometheus_alert:
+              namespace_prefix: "app-a"
+      actions:
```

Comment:
Is the indentation on this correct? Shouldn't this be aligned with triggers?

Reply:
fixed